src/features/chains/utils.ts

```diff
@@ -31,3 +31,26 @@ export function getChainByRpcUrl(multiProvider: MultiProtocolProvider, url?: str
     (m) => !!m.rpcUrls.find((rpc) => rpc.http.toLowerCase().includes(url.toLowerCase())),
   );
 }
+
+/**
+ * Return a customListItemField object that contains the amount of
+ * routes from a single chain to every other chain
+ */
+export function getCustomListItemField(
```

Comment:
Please move this to ChainSelectField as that's the only place that it will be used.

Reply:
kept in here but renamed to `getAllTokenRoutesFromChain`, please lmk what you think